tetris.py

```diff
@@ -370,4 +316,28 @@ def full_line_clear(self):
                     ] + [(a, b + 1) for (a, b) in coord_list if b < x_line]
 
 
+
+class TetrisGUI:
+    def __init__(self, speed, main_board):
+        self.speed = speed
```

Comment:
Speed argument and `self.speed` attribute are never used. Delete them.

Reply:
Deleted, but I think we should reintroduce this in the future for Board class

Reply:
This hasn't gotten deleted either